### PR TITLE
Generate docs

### PR DESCRIPTION
**Pull request by hashtagchris**
I noticed vsts-task-lib.json had `publishTelemetry`, but that function was reverted.

### CODE REVIEW

**Comment by stephenmichaelf** [comment]
Is this the results of running gendocs?

**Comment by hashtagchris** [comment]
Yup. After I saw the changes pended to node/docs/vsts-task-lib.json, I edited node/docs/docs.json by hand to add more exported methods, and ran gendocs again. run.sh gave me trouble, so I ran node gendocs.js directly.

File: node/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vsts-task-lib",
-  "version": "2.5.0",
+  "version": "2.6.0",
```
Comment (hashtagchris) [inline node/package-lock.json]: I think npm install pended this. Now the file is consistent with package.json.

**Comment by stephenmichaelf** [comment]
Looks good, any other changes or ready to merge?

**Comment by hashtagchris** [comment]
That's all I have planned. It would be good to run the tool periodically.

**Pull request merged**